src/stan/lang/grammars/term_grammar_def.hpp

```diff
@@ -231,15 +231,17 @@ namespace stan {
         %= lexeme[char_("a-zA-Z")
                   >> *char_("a-zA-Z0-9_.")];
 
-      prob_args_r.name("probability function arguments");
+      prob_args_r.name("probability function argument");
       prob_args_r
         %= (lit('(') >> lit(')'))
         | hold[lit('(')
                >> expression_g(_r1)
                >> lit(')')]
         | (lit('(')
            >> expression_g(_r1)
-           >> lit('|')
+           >> (lit(',')
```

Comment:
@bob-carpenter I'm reading this as triggering the new error message function if you run into a literal ',' after `expression_g` consumes the first function argument or continuing if it gets a literal '|'?